app/models/claim.rb

```diff
@@ -531,10 +531,6 @@ def has_ecp_policy?
     policy == Policies::EarlyCareerPayments
   end
 
-  def has_tslr_policy?
-    policy == Policies::StudentLoans
-  end
-
```

Comment:
mmmh I am using this method here https://github.com/DFE-Digital/claim-additional-payments-for-teaching/blob/ccab16afbab9c3bebda50ecec77c35f72a141289/app/models/claim_student_loan_details_updater.rb#L12

Reply:
Thanks, I will remove this commit in that case.